Dockerfile.composer

```diff
@@ -0,0 +1,15 @@
+FROM composer:1.6.4
+
+ARG composer_dev_arg
+
+COPY composer.json \
+    composer.lock \
+    ./
+
+RUN composer --no-interaction install ${composer_dev_arg} --ignore-platform-reqs --no-autoloader --no-scripts --no-suggest
+
+COPY test/ test/
+COPY src/ src/
+
+RUN composer --no-interaction dump-autoload ${composer_dev_arg} --classmap-authoritative
+RUN composer --no-interaction run-script post-install-cmd
```

Comment:
it's interesting to see that even the most complex `composer` image is quite consistent